### PR TITLE
DEPRECATED

### PR DESCRIPTION
**Pull request by pavolloffay**
## Dot not merge this is only for grammer review!

current pdf: https://github.com/hawkular/hawkular-datamining/raw/doc/doc/thesis.pdf

### CODE REVIEW

File: doc/content.tex [hunk 1]
```diff
@@ -1,1401 +0,0 @@
...
-    The problem was that Inventory is developed by another party and any concrete document
-    describing the integration was not proposed. The design of the integration was understood differently
-    by both parties, which lead to reimplementing of some parts. Therefore it is better to precisely define the
-    integration interface with other modules or projects.
-
-%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\chapter{Evaluation of Implemented Models} \label{chap:evaluation}
-Evaluation process compares predictive capabilities of the implemented models to statistics produced by similar models
-from R language. Models from R language are selected because they are widely used by forecasters all around the
-world and de facto standard for statistical science \cite{kleiber2008applied}, therefore
```
Comment (grammar-review) [inline doc/content.tex:1065]: test

Comment (karelhala) [inline doc/content.tex]: Asdfghjkkl

Comment (pavolloffay) [inline doc/content.tex:1065]: komentar

File: doc/content.tex [hunk 2]
```diff
@@ -1,1401 +0,0 @@
...
-    All tests are automatically executed every time developer creates a pull request or pushes code to the master
-    branch. For this Travis continuous integration platform was used, which is available for free for open source
-    projects on Github.
-
-    Documentation is directly written in Java code as javadoc. REST API documentation is also directly
-    written in source code using Swagger framework and automatically generated at build time
-    \footnote{Available at \url{http://www.hawkular.org/docs/rest/rest-datamining.html}.}.
-
-    %%%%%%%%
-    \section{Releases}
```
Comment (karelhala) [inline doc/content.tex]: tyrty

File: doc/content.tex (hunk 1 above)
Comment (karelhala) [inline doc/content.tex]: uuuu

File: doc/thesis.tex [hunk 3]
```diff
@@ -1,120 +0,0 @@
...
-\FrontMatter                    % The front matter
-\ThesisTitlePage                % The title page
-\begin{ThesisDeclaration}       % The declaration
-  \DeclarationText
-  \AdvisorName
-\end{ThesisDeclaration}
-
-\begin{ThesisThanks}            % The acknowledgements (optional)
-  I would like to thank my supervisor RNDr. Adam Rambousek, Ph.D., Mgr. Jiri Kremser,
-  family and the people from Hawkular team. Next, my thanks goes to Ing. Daniel Němec, Ph.D.
```
Comment (karelhala) [inline doc/thesis.tex]: ghjghjj

**Pull request closed without merging**